lib/establish_connection.py

```diff
@@ -81,6 +81,22 @@ def get_input_params():
         default=False,
         dest='deprecate_rest',
         help='/api and /rest rendering - with /rest being deprecated')
+    parser.add_argument(
+        '-fam',
+        '--fetch-authentication-metadata',
+        required=False,
```

Comment:
Isn't True a better default value (as the generated OpenAPI definition will be contain the security aspect)?